web/css/main.css

```diff
@@ -0,0 +1,10 @@
+﻿/*!
```

Comment:
Should this file really be added to the repo? I could imagine that it will get really messy with different people committing this file…

And as it will be rendered newly during deployment the deployed file will *never* resemble the one checked in…